exercises/raindrops/uRaindropsTest.pas

```diff
@@ -0,0 +1,179 @@
+unit uRaindropsTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  RaindropsTest = class(TObject)
+  public
+    [Test]
+    procedure the_sound_for_1_is_1;
+
+    [Test]
+    [Ignore('Comment these lines to run these tests')]
```

Comment:
I think it should read...

`// [Ignore('Comment the "[Ignore]" statement to run the test')]` and it reinforces that it has to be done individually, as there is generally only one ignore statement per test.